### PR TITLE
Hosting Config: Enable current Cache Clear logic for all users

### PR DESCRIPTION
**Pull request by wojtekn**
Related to 1476-gh-Automattic/dotcom-forge

## Proposed Changes

In this PR, I propose to enable all users to clear the cache for their Atomic sites and to refresh the whole block.

image

## Testing Instructions

1. Navigate to Atomic site, Settings -> Hosting Configuration 
2. Locate the Cache block at the bottom of the page
3. Clear cache
4. Confirm cache was cleared, that button is disabled, and informative text appears
5. Wait a minute and confirm that button is enabled again

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/state/selectors/should-rate-limit-atomic-cache-clear.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'calypso/state/hosting/init';
+
+const FIVE_MINUTES_IN_MILLISECONDS = 300 * 1000;
```
Comment (danielbachhuber) [inline client/state/selectors/should-rate-limit-atomic-cache-clear.js]: I think a minute is fine. Five minutes simply adds unnecessary delay to the user's workflow.

**Review by danielbachhuber — APPROVED** [review]
✨ 

@wojtekn FYI - I added `Hosting Config:` prefix to your pull request title, as well as a screenshot.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7817398

Thank you @wojtekn for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7817398).

**Pull request merged**